### DISCUSSION TITLE
Revoke Method

### DISCUSSION THREAD

**Discussion by ghost** [njoguamos/laravel-zoho-oauth#23] [General]
Hi,

How soon would the revoke method be ready? Also, do you have any idea how OAuth 2 implementation would look like?

Thanks

**Comment by njoguamos** [comment]
>How soon would the revoke method be ready?

I will take some time and implement the revoke method within the next 7 days.

>Also, do you have any idea how OAuth 2 implementation would look like?

Are you still referring to token revocation or the entire package?

**Reply by ghost** [reply]
I am looking at a scenario where browser-based oauth 2 authentication would be ideal especially when a user is on shared hosting and doesn't have access to shell/terminal.

1. How long it would take to tinker around with the OAuth 2 implementation
2. How long to implement the revoke method — but you have said 7 days

**Reply by njoguamos** [reply]
Unfortunately, I am not conversant with hosting Laravel applications on shared hosting.

Implementing the browser-based OAuth 2 authentication is out of the scope of this package. If you have some ideas, you can share them.

Let me know if this can help https://github.com/SocialiteProviders/Zoho

**Reply by ghost** [reply]
Circling back to check if there has been an update on the``revoke`` method
